certified-connectors/AIForged/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4627 @@
+          {
+            "type": "integer",
+            "name": "id",
+            "in": "query",
+            "description": "Document id to delete",
+            "format": "int32",
+            "x-nullable": false,
+            "x-ms-summary": "Document id to delete"
+          },
+          {
+            "type": "string",
+            "name": "X-Api-Version",
+            "in": "header",
+            "required": true,
+            "description": "API version header",
+            "x-nullable": true,
+            "x-ms-summary": "API version header"
+          }
+        ],
+        "responses": {
+          "200": {
+            "x-nullable": false,
+            "description": "The deleted document",
+            "schema": {
+              "$ref": "#/definitions/AIForged.ViewModels.DocumentViewModel"
+            }
+          }
+        }
+      }
+    },
+    "/api/Document/GetPreviews": {
+      "get": {
+        "tags": [
+          "Document"
+        ],
+        "summary": "Get preview documents and images for a service",
+        "description": "Get preview documents and images for a service",
+        "operationId": "Document_GetPreviews",
+        "produces": [
+          "text/plain",
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "type": "string",
+            "name": "userId",
+            "in": "query",
+            "description": "The user that owns the service",
+            "x-nullable": true,
+            "x-ms-summary": "The user that owns the service"
+          },
+          {
+            "type": "integer",
+            "name": "projectId",
+            "in": "query",
+            "description": "The project id",
+            "format": "int32",
+            "x-nullable": false,
+            "x-ms-summary": "The project id"
+          },
+          {
+            "type": "integer",
+            "name": "stpdId",
+            "in": "query",
+            "description": "The service id",
+            "format": "int32",
+            "x-nullable": true,
+            "x-ms-summary": "The service id"
+          },
+          {
+            "type": "string",
+            "name": "X-Api-Version",
+            "in": "header",
+            "required": true,
+            "description": "API version header",
+            "x-nullable": true,
+            "x-ms-summary": "API version header"
+          }
+        ],
+        "responses": {
+          "200": {
+            "x-nullable": false,
+            "description": "The preview document",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/AIForged.ViewModels.DocumentViewModel"
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api/Document/GetExtended": {
+      "get": {
+        "tags": [
+          "Document"
+        ],
+        "summary": "Find documnets by using structured search criteria\nThis results can be paged",
```

Comment:
@aiforged Please remove html tags from the summary tags, it may not be supported in the Power platform pages.